backend/app.js

```diff
@@ -1,24 +1,26 @@
 const express = require('express');
```

Comment:
Something that should immediately cross your mind when seeing a package.json  should be adding node_modules to the gitignore file